src/utils.jl

```diff
@@ -300,7 +300,7 @@ midpoint(cell::Ferrite.AbstractCell{3,N,6}, points) where N = Point3f(0.5 * (poi
 function postprocess(node_values)
     dim = length(node_values)
     if dim == 1
-        return node_values
+        return node_values[1]
```

Comment:
here or in the source code? 
The function was inherently type unstable because if the dimension of the input is 1 it returned a vector of length 1 and in all other cases it reduced the vector entries to a scalar value

Reply:
In the source code (e.g. for contributors seeing this).